### PR TITLE
graphics/drawio: Update to 22.1.2 and disable auto updater on first run

### PR DESCRIPTION
**Pull request by Zoid-dr**
- disable auto updater on first run
- update to 22.1.2

### CODE REVIEW

**Comment by rosorio** [comment]
looks good to me, I take it.

File: graphics/drawio/Makefile [hunk 1]
```diff
@@ -1,12 +1,11 @@
 PORTNAME=	drawio
-PORTVERSION=	22.0.3
+PORTVERSION=	22.1.2
 DISTVERSIONPREFIX=	x86_64-
-PORTREVISION=	1
 CATEGORIES=	graphics
 MASTER_SITES=	https://github.com/jgraph/drawio-desktop/releases/download/v${PORTVERSION}/
 EXTRACT_SUFX=	.rpm
 
-MAINTAINER=	636808@mail.ru
```
Comment (rosorio) [inline graphics/drawio/Makefile:9]: You didn't mention you update the maintainer email. Any particular reason ?

Comment (Zoid-dr) [reply graphics/drawio/Makefile:9]: I have updated the email because I am having problems with my mail.ru-hosted email.

Comment (rosorio) [reply graphics/drawio/Makefile:9]: ok, thanks for the explanation :+1:

**Comment by rosorio** [comment]
merged as 697a94b55c01269fcc30436c47fa42dac26a65be

**Pull request closed without merging**